websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,108 @@
+const presence = new Presence({
+		clientId: "981442534167441428"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, hostname } = document.location;
+
+	if (["www.brain.fm", "try.brain.fm"].includes(hostname)) {
+		if (["/", "/home"].includes(pathname))
+			presenceData.details = "Viewing the home page";
+		else if (pathname.startsWith("/about"))
+			presenceData.details = "Viewing the about page";
+		else if (pathname.startsWith("/pdfs"))
+			presenceData.details = "Reading a PDF";
```

Comment:
can you show which PDF?

Reply:
The white papers on the bottom of the science page:
https://www.brain.fm/science
https://www.brain.fm/pdfs/performance-pilot.pdf
https://www.brain.fm/pdfs/white-paper.pdf
https://www.brain.fm/pdfs/sleep-study.pdf
![image](https://user-images.githubusercontent.com/49926644/172263288-b819afab-c78b-4d84-af21-264978ece7bf.png)

Reply:
I misunderstood what you meant. I have implemented that now.